prisma/schema.prisma

```diff
@@ -246,8 +246,10 @@ model NewApplication {
   disabilityCertificationDate          DateTime                             @map("disability_certification_date") @db.Date
   patientCondition                     PatientCondition                     @map("patient_condition")
   mobilityAids                         MobilityAid[]                        @map("mobility_aids")
+  otherMobilityAids                    String?                              @map("other_mobility_aids") @db.VarChar(255)
```

Comment:
Added `otherMobilityAids` to match the design

<img width="563" alt="image" src="https://user-images.githubusercontent.com/51224641/167759220-49db742d-a02a-4ae6-b112-b111efdd71c6.png">